spreadsheet/chapter.md

```diff
@@ -307,7 +307,7 @@ With these properties in place, we can define the `calc()` function that trigger
     const json = angular.toJson( $scope.sheet );
 ```
 
-Here we first snapshot the state of `sheet` into a JSON string, stored in the constant `json`.
+Here we first take a snapshot of the state of `sheet` and store it in the constant `json`, a JSON string.
```

Comment:
I rewrote this to avoid the use of "snapshot" as a verb -- please check to make sure I didn't change your meaning by mistake.